src/script/redux/reducers.js

```diff
@@ -0,0 +1,7 @@
+const { simpleObject } = require('trampss-redux-factory')
+
+module.exports = {
+  component: simpleObject('component'),
+  project: simpleObject('project'),
+  config: simpleObject('config'),
```

Comment:
here we could set `defaultData` with `redux-factory` to have a description of the waiting data in each reducers